gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -14,8 +14,17 @@
               <input ng-if="editingDeviceNew()" name="deviceID" id="deviceID" class="form-control text-monospace" type="text" ng-model="currentDevice.deviceID" required="" valid-deviceid list="discovery-list" aria-required="true" />
               <div ng-if="!editingDeviceNew()" class="well well-sm form-control text-monospace" style="height: auto;" select-on-click>{{currentDevice.deviceID}}</div>
               <div class="input-group-btn">
```

Comment:
You could experiment with `input-group-addon` as it is used in `editShareTemplate.html` for the lock and eye indicators.  I think it looks more consistent especially with the dark theme.  Also please check if a separate `input-group-btn` or `input-group-addon` per button makes sense.  The `editShareTemplate.html` directive e.g. uses `<span>`, dunno if that makes a difference.

Reply:
It's a late reply, but I've just checked those. They're not great actually, as the `div`s and `span`s can't be focused with keyboard. Buttons, on the other hand, can, so they're much better in terms of accessibility in this regard.